aiida/cmdline/commands/cmd_status.py

```diff
@@ -132,13 +132,14 @@ def verdi_status(print_traceback, no_rmq):
             with Capturing(capture_stderr=True):
                 with override_log_level():  # temporarily suppress noisy logging
                     comm = manager.create_communicator(with_orm=False)
-                    comm.close()
         except Exception as exc:
             message = f'Unable to connect to rabbitmq with URL: {profile.get_rmq_url()}'
             print_status(ServiceStatus.ERROR, 'rabbitmq', message, exception=exc, print_traceback=print_traceback)
             exit_code = ExitCode.CRITICAL
         else:
             print_status(ServiceStatus.UP, 'rabbitmq', f'Connected as {profile.get_rmq_url()}')
+        finally:
+            comm.close()
```

Comment:
Can this fail if comm is not defined (e.g. if `Capturing` or `override_log_level` raises)?

Reply:
Yes, it would indeed raise a `NameError`. Will fix it, but maybe that is exactly why we don't need the `finally` clause. If either the context managers except, _or_ `get_communicator` excepts, then `comm` will not be defined and so there is no need to close it, so I think I can just revert this change and leave the `close()` call in the `try`